src/messages.js

```diff
@@ -683,13 +683,12 @@ function resolveHelpRequestBlocks({thread_ts}) {
                         "emoji": true
                     },
                     "options": [
-                        option('Infrastructure issue'),
-                        option('Lack of documentation'),
+                        option('External (Github/Azure/Sonacloud) issue'),
```

Comment:
```suggestion
                        option('External (GitHub/Azure/SonarCloud) issue'),
```

Reply:
Yes I tested this and it works ok - see https://tools.hmcts.net/jira/browse/SBOX-310

Reply:
Typo corrected:

option('External (GitHub/Azure/SonarCloud) issue'),